build.gradle

```diff
@@ -119,83 +100,57 @@ minecraft {
             }
         }
 
+        client {
+            workingDirectory project.file("run/client")
```

Comment:
why not just run?

Reply:
Thought was so it's clear we have run/client, run/server, and run/gametest. i.e. any individual files belong to either client, server, or gametest (no files are just in run, not a priori knowing what process they came from)